README.md

```diff
@@ -1,28 +1,51 @@
-
 Node.js
-=====
+=======
 
 [![Gitter](https://badges.gitter.im/Join Chat.svg)](https://gitter.im/nodejs/node?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-This repository began as a GitHub fork of
-[joyent/node](https://github.com/joyent/node).
-
-Node.js contributions, releases, and contributorship are under an
-[open governance model](./GOVERNANCE.md).
-We intend to land, with increasing regularity, releases which are
-compatible with the npm ecosystem that has been built to date for
-Node.js.
-
-We also have a Code of Conduct. [Please read it.](./CODE_OF_CONDUCT.md)
+The Node.js project is supported by the
+[Node.js Foundation](https://nodejs.org/en/foundation/). Contributions,
+policies and releases are managed under an
+[open governance model](./GOVERNANCE.md). We are also bound by a
+[Code of Conduct](./CODE_OF_CONDUCT.md).
+
+## Release Types
+
+The Node.js project maintains multiple types of releases:
+
+* **Stable**: planned and signed releases. Released from active development
+  branches of this repository and versioned by [SemVer](http://semver.org/).
+  Code for Stable releases are organized in this repository by major version
+  number, For example:. [v4.x](https://github.com/nodejs/node/tree/v4.x).
+  The major version number of Stable releases will increment every 6 months
+  allowing for breaking changes to be introduced. This happens in April and
+  October every year. Stable release lines beginning in October each year have
+  a maximum support life of 8 months. Stable release lines beginning in April
+  each year will convert to LTS (see below) after 6 months and receive further
+  support for 30 months.
+* **LTS**: Every second Stable release line (major version) will become LTS
```

Comment:
Micro-nit: The other two entries (Stable and Nightly) each start with a sentence fragment and no capitalization but this one starts with a full sentence properly capitalized. In a world where you had nothing better to do than to write the perfect README, they'd all have the same format.